lib/melissa_data/geo_lookup/geocoder.rb

```diff
@@ -14,9 +14,18 @@ def address_to_coordinates(address)
       end
 
       def coordinates?(response)
-        lat = response[:property_address][:latitude]
-        long = response[:property_address][:longitude]
-        lat != nil && long != nil
+        if !response[:errors]
+          lat = response.fetch([:property_address])[:latitude]
+          long =response.fetch([:property_address])[:longitude]
+          lat != nil && long != nil
```

Comment:
For a second I thought this could be simply:

```
lat && long
```
Since they would be be assigned to nil, but in IRB:

```ruby
irb> a = {key: "val"}
irb> lat = a[:lat]
irb> long = a[:long]
irb> lat && long
# => nil
```